mk-files/release-notes.mk

```diff
@@ -8,31 +8,27 @@ release-notes:
 	bump=$$(cat release-notes/bump.txt) && \
 	version=$$(cat release-notes/version.txt) && \
 	cd $(CONFLUENT_DOCS_DIR) && \
-	git fetch && \
 	if [ "$${bump}" = "patch" ]; then \
-		git checkout $$(echo $${version} | sed $(STAGING_BRANCH_REGEX)) ; \
+		git checkout $$(echo $${version} | sed $(STAGING_BRANCH_REGEX)); \
 	fi && \
 	git checkout -b cli-v$${version}-release-notes && \
-	CONFLUENT_DOCS_DIR=$(CONFLUENT_DOCS_DIR) make publish-release-notes-to-docs-repo
-
+	cd - && \
```

Comment:
why is this?

Reply:
`make publish-release-notes-to-docs-repo` can only be run from the CLI repo, not the docs repo